.golangci.yaml

```diff
@@ -146,10 +138,4 @@ issues:
     - path: pkg/golinters/unused.go
       text: "rangeValCopy: each iteration copies 160 bytes \\(consider pointers or indexing\\)"
 
-run:
```

Comment:
Why was this removed?

Reply:
The same issue with Golang lint, it keeps giving warning . Bot of those files keeps giving warnings